tembo-cli/src/cmd/login.rs

```diff
@@ -22,17 +22,21 @@ struct SharedState {
 }
 
 #[derive(Args)]
-#[clap(author, version, about, long_about = None)]
+#[clap(author, version, about = "Initiates login sequence to authenticate with Tembo", long_about = None)]
```

Comment:
this section, same wording from this other comment https://github.com/tembo-io/tembo/pull/703#pullrequestreview-1991749109